PyInstaller/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 # Note: Keep this variable as plain string so it could be updated automatically
 #       when doing a release.
-__version__ = '3.4.dev0'
+__version__ = '3.4.dev5'
```

Comment:
Please don't change the version number in a pull-request.

Reply:
Right, sorry about that, should have created a branch for the pull request. I've been pushing changes that I need, and building my own pyinstaller version. I'll try to fix that asap.